ballerina/Module.md

```diff
@@ -21,5 +21,3 @@ Uses a pseudo-random number generator to generate the UUID. Every bit of the str
 Similar to Version 3 but uses SHA-1 instead of MD5.
 
 Other operations include validating a given UUID string and getting the version of a UUID string.
-
-For information on the operations, which you can perform with the `uuid` module, see the below **Functions**.
```

Comment:
Shall we keep these lines in module.md

Reply:
Update in [dcd7453](https://github.com/ballerina-platform/module-ballerina-uuid/pull/362/commits/dcd74535d86bec7c7f1558727e4c328a69ea70d1)